app/models/concerns/hyrax/solr_document/metadata.rb

```diff
@@ -71,6 +71,7 @@ def self.coerce(input)
         attribute :workflow_state, Solr::String, solr_name('workflow_state_name', :symbol)
         attribute :human_readable_type, Solr::String, solr_name('human_readable_type', :stored_searchable)
         attribute :representative_id, Solr::String, solr_name('hasRelatedMediaFragment', :symbol)
+        attribute :rendering_ids, Solr::Array, solr_name('hasFormat', :symbol)
```

Comment:
Is this dependent on `Hyrax.config.rendering_predicate`?

Reply:
Changed so that it retrieves the name from the config.